<launch>

<arg name="gui" default="False"/>


<param  name="robot_description"
        command="$(find xacro)/xacro --inorder '$(find open_base)/xacro/main.urdf.xacro'" />

<!-- <node name="driver_node_a" pkg="roboteq_python" type="roboteq_driver_node" output="screen">
    <rosparam command="load" file="$(find omnicone)/config/motors_a.yaml" />
</node>

<node name="driver_node_b" pkg="roboteq_python" type="roboteq_driver_node" output="screen">
    <rosparam command="load" filse="$(find omnicone)/config/motors_b.yaml" />
</node> -->

<node name="kinematics_node" pkg="open_base" type="open_base_kinematics" output="screen"/>

<!-- <node name="vel_command_publisher" pkg="omnicone" type="vel_command_publisher" output="screen"/> -->

<node name="Waypoint" pkg="omnicone" type="Waypoint.py" output="screen"/>

<!-- <node name="Kinematic_Controller" pkg="omnicone" type="Kinematic_Controller.py" output="screen"/> -->

<node name="EKF" pkg="omnicone" type="EKF.py" output="screen"/>


</launch>
